test/query_comments_test.rb

```diff
@@ -221,10 +221,14 @@ def test_hostname_and_pid
     assert_match %r{/\*hostname:#{Socket.gethostname},pid:#{Process.pid}\*/$}, @queries.first
   end
 
+  def test_controller_with_single_level
+    PostsController.action(:driver_only).call(@env)
+    assert_match %r{,?controller:posts,?}, @queries.first
+  end
+
   def test_controller_with_namespace
-    Marginalia::Comment.components = [:controller_with_namespace]
     API::V1::PostsController.action(:driver_only).call(@env)
-    assert_match %r{/\*controller_with_namespace:API::V1::PostsController}, @queries.first
+    assert_match %r{,?controller:api::v1::posts,?}, @queries.first
```

Comment:
I tightened this up to allow zero or more `,` to denote the start and end of the component section. Unsure how the previous one was matching 🤷‍♂